maestrowf/maestro.py

```diff
@@ -319,6 +319,9 @@ def setup_argparser():
     run.add_argument("-fg", action="store_true", default=False,
                      help="Runs the backend conductor in the foreground "
                      "instead of using nohup. [Default: %(default)s]")
+    run.add_argument("--hashws", action="store_true", default=False,
+                     help="Enable hashing of subdirectories in parameterized "
```

Comment:
Consider adding a note about the use of parameter label attributes in a study when `--hashws` is specified.